package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@api3/ois": "^2.3.2",
     "@api3/promise-utils": "^0.4.0",
-    "@octokit/rest": "^21.0.2",
+    "@octokit/rest": "^20.1.1",
```

Comment:
This should be set as ignored update similarly how it's done [here](https://github.com/api3dao/oev-searcher/blob/980696dea8973c9327144db8a9f286114857c549/renovate.json#L6).

Reply:
Thanks for catching this. I completely missed it before